### DISCUSSION TITLE
Discharged/Charged Energy over modbus

### DISCUSSION THREAD

**Discussion by ctyd** [Louisvdw/dbus-serialbattery#208] [Ideas]
Hi,

is it Possible to readout the Discharged and charged Energy? actually i have a JBD Bms with your driver on venusos, all runs fine. my Smarthome (homeassistant) is connected to venus over modbus. i can readout the two sensors over modbus but there are no values in the sensors.

**Comment by Louisvdw** [comment]
Most BMS do not have a history of the energy available. The JBD is one of those.
But I guess we can estimate it with 
`battery capacity * cycles`
